src/OpenTelemetry.Contrib.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog - OpenTelemetry.Contrib.Instrumentation.AWS
 
+## 1.0.2
+
+Released 2022-Oct-21
```

Comment:
We don't yet know when this will be released. Let's just put this change under an `Unreleased` heading.

Reply:
Done